react-state/my-app/src/ToggleButton.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react"
+export default function ToggleButton ({text, color}) {
+  const [isClicked, setisClicked] = useState(false)
+  const handleClick = () => {
+    setisClicked(!isClicked);
+    console.log(isClicked);
```

Comment:
Log 3 times so you can see how it changes:
1. During render (in the main body of the ToggleButton function)
2. Just before calling `setIsClicked`
3. Just after calling `setIsClicked`